doc/theory/glossary.tex

```diff
@@ -16,56 +70,254 @@
 	
-	\paragraph{Filtered simplicial complex:} A \textit{filtered simplicial complex} is a collection of simplicial complexes $\{X(n)\}_{n \geq 0}$ such that $X(n)$ is a subcomplex of $X(n+1)$ for each $n \geq 0$. A filtered simplicial complex such that the difference between $X(n+1)$ and $X(n)$ is exactly one simplex is called a \textit{simplexwise filtration}. The data of a simplexwise filtration is equivalent to a simplicial complex $X$ together with a total order $\leq$ on its simplices such that for each $y \in X$ the set $\{x \in X\ :\ x \leq y\}$ is a subcomplex of $X$.
+	A \underline{simplicial map} induces a \underline{chain map} between the associated simplicial chain complexes and, therefore, between the associated simplicial (co)homologies.
 	
-	\paragraph{Persistence modules and barcodes:} A \textit{persistence module} is a collection of vector spaces $\{V(n)\}_{n \geq 0}$ together with linear maps $V(n) \to V(n+1)$. We now define an invariant that completely characterizes a persistence module.  A \textit{multiset} is a pair $(S, \phi)$ where $S$ is a set and  $\phi : S \to \mathbb Z$ is a function attaining positive values. For $s \in S$ we refer to $\phi(s)$ as its \textit{multiplicity}. The \textit{barcode} of a persistence module $V$ is the multiset of half-open intervals $[a,b) \subset \mathbb R$ such that the image of the composition $V(i) \to \cdots \to V(j)$ has dimension equal to the number, counted with multiplicity, of half-open intervals containing $i$ and $j$.
+	\subsection*{Cubical chains and cubical homology:} Let $X$ be a cubical complex. Define its \textit{cubical chain complex with $\Bbbk$-coefficients} $C_*(X; \Bbbk)$ by 
+	\begin{equation*}
+	C_n(X; \Bbbk) = \Bbbk\{X_n\} \qquad \partial_n x = \sum_{i = 1}^{n} (-1)^{i-1}(d^+_i x - d^-_i x)
+	\end{equation*}
+	where $x = I_1 \times \cdots \times I_N$ and $s(i)$ is the dimension of $I_1 \times \cdots \times I_i$.
+	Its \textit{homology and cohomology with $\Bbbk$-coefficients} is the \underline{homology and cohomology} of this chain complex. We use the notation $H_*(X; \Bbbk)$ and $H^*(X; \Bbbk)$ for these.
 	
-	\section{Geometry}
+	\subsection*{Filtered complex:}
+	A \textit{filtered complex} is a collection of simplicial or of cubical complexes $\{X(n)\}_{n \in \mathbb N}$ such that $X(n)$ is a subcomplex of $X(n+1)$ for each $n \geq 0$.
 	
-	\paragraph{Metric space:} A  pair $(X, d)$ where $X$ is a set and $d$ is a function 
+	\subsection*{Cellwise filtration:}
+	A \underline{filtered complex} such that $X(n+1)$ contains exactly one more simplex or elementary cube than $X(n)$. \par
+	The data of a simplexwise filtration is equivalent to a complex $X$ together with a total order $\leq$ on its simplices or elementary cubes such that for each $y \in X$ the set $\{x \in X\ :\ x \leq y\}$ is a subcomplex of $X$.
```

Comment:
Cellwise filtration sounds good. Can you incorporate this term in this sentence as well?

Reply:
sure